vnext/Microsoft.ReactNative/Utils/LocalBundleReader.cpp

```diff
@@ -15,6 +15,44 @@
 
 namespace Microsoft::ReactNative {
 
+std::string GetBundleFromEmbeddedResource(winrt::hstring str) {
```

Comment:
Where is this used/tested? Shouldn't we add the project-related logic to either an existing unpackaged repo app or at least the WinAppSDK template, in release mode?

Reply:
right now I'm only testing it locally; we don't have an unpackaged winappsdk app checked in yet.